### DISCUSSION TITLE
Dreame D9: no controls over the mop mode anymore

### DISCUSSION THREAD

**Discussion by Sauberkiwi** [merdok/homebridge-miot#310] [Q&A] [answered]
Hey I got an dreame d9 connected to hk via hoobs and miot. Anything worked well so far until my card got corrupted. After a reflash (and no backup) all my plugins are gone. So I reinstalled miot and connected my dreame. But now, I got no controls over the mop mode anymore. Can anyone help me with this issue? It’s important because I use Automations that toggle she mop water mode.

Regards,
Sauberkiwi

**Comment by merdok** [comment, marked as answer]
You probably need to add some `actionButtons` or `propertyControl`. Just lookup your device here and you will see what id's you have to use: http://merdok.org/miotspec
